src/components/Header/header.jsx

```diff
@@ -1,67 +1,132 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { Link } from 'react-router-dom';
+import { Header } from '@master-c8/commons';

```

Comment:
Me genera un error al compilar: `export 'Header' (imported as 'Header') was not found in '@master-c8/commons' (possible exports: Logotype, __esModule)`

Reply:
Si, debes instalar los paquetes de nuevo

Reply:
Fixed, gracias